.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/firesim/aws-fpga-firesim
 [submodule "target-design/chipyard"]
 	path = target-design/chipyard
-	url = https://github.com/ucb-bar/chipyard
+	url = https://github.com/sifive/chipyard
```

Comment:
Shouldn't merge with this in place.  Wasn't sure how to get the CI to work on the PR without pointing to my fork of CY since I don't have write access to the ucb-bar fork.  Since "pull request authors can't request changes on their own PR", (nice feature bro), I'm going to add the DO-NOT-MERGE label on this until I revert the chipyard url change.

Reply:
Removed after realizing that as long as I have a PR open on ucb-bar/chipyard with the commit, it will be available from the upstream chipyard.